packages/formation/js/accordion.js

```diff
@@ -16,42 +20,45 @@ const isElementInViewport = (
 };
 
 const addAriasExpandedAttr = () => {
-  document.querySelectorAll('.usa-accordion-button').forEach(element => {
-    const hasAriasExpandedAttr = element.getAttribute('aria-expanded');
+  Array.from(document.querySelectorAll('.usa-accordion-button')).forEach(
+    element => {
+      const hasAriasExpandedAttr = element.getAttribute('aria-expanded');
 
-    if (!hasAriasExpandedAttr) {
-      element.setAttribute('aria-expanded', false);
-    }
-  });
+      if (!hasAriasExpandedAttr) {
+        element.setAttribute('aria-expanded', false);
+      }
+    },
+  );
 };
 
 const addAriaHiddenAttr = () => {
-  document.querySelectorAll('.usa-accordion-content').forEach(el => {
-    const buttonElement = document.querySelector(`[aria-controls="${el.id}"]`);
-
-    if (buttonElement) {
-      const hiddenValue = !toBoolean(
-        buttonElement.getAttribute('aria-expanded'),
+  Array.from(document.querySelectorAll('.usa-accordion-content')).forEach(
+    el => {
+      const buttonElement = document.querySelector(
+        `[aria-controls="${el.id}"]`,
       );
 
-      el.setAttribute('aria-hidden', hiddenValue);
-    }
-  });
+      if (buttonElement) {
+        const hiddenValue = !toBoolean(
+          buttonElement.getAttribute('aria-expanded'),
+        );
+
+        el.setAttribute('aria-hidden', hiddenValue);
+      }
+    },
+  );
 };
 
-const getParents = (elem, selector) => {
+const getAccordionParents = elem => {
```

Comment:
I updated this to avoid having to polyfill `matches`, though now that I've polyfilled `closest`, we could switch back.